frontend/src/components/object/ObjectTag.vue

```diff
@@ -98,19 +99,18 @@ watch( [props, tsGetTagging, vsGetTagging], () => {
       v-for="tag in objectTagging?.tagset"
       :key="tag.key + tag.value"
     >
-      <div class="col">
-        <Button
-          label="Primary"
-          class="p-button-raised p-button-rounded"
-        >
-          {{ tag.key + "=" + tag.value }}
-        </Button>
+      <div class="grid">
+        <div class="col mr-2">
+          <Tag value="Primary" rounded>
```

Comment:
Linting
```ts
<Tag 
    value="Primary"
    rounded
>
```
We'll need to revisit the linting rules at some point, but for now should get this formatted so that eslint doesn't keep complaining.